File: JSBASIC/105_restaurant_menu.js
// 105	105_restaurant_menu.js	Create a restaurant menu system — show item and price using switch.

let item = 3;
console.log("=== Restaurant Menu ===");
console.log("1. Butter Chicken  - ₹220");
console.log("2. Paneer Tikka    - ₹180");
console.log("3. Dal Makhani     - ₹150");
console.log("4. Biryani         - ₹260");
console.log("5. Gulab Jamun     - ₹80");
console.log("=======================");
console.log("Your choice:", item);
switch (item) {
    case 1: console.log("You ordered: Butter Chicken - ₹220"); break;
    case 2: console.log("You ordered: Paneer Tikka - ₹180"); break;
    case 3: console.log("You ordered: Dal Makhani - ₹150"); break;
    case 4: console.log("You ordered: Biryani - ₹260"); break;
    case 5: console.log("You ordered: Gulab Jamun - ₹80"); break;
    default: console.log("Invalid choice.");
}

File: JavaBasic/105_restaurant_menu.java
// 105	105_restaurant_menu.java	Create a restaurant menu system — show item and price using switch.
public class _105_restaurant_menu {
    public static void main(String[] args) {
        int item = 3;
        System.out.println("1. Butter Chicken - Rs.220");
        System.out.println("2. Paneer Tikka   - Rs.180");
        System.out.println("3. Dal Makhani    - Rs.150");
        System.out.println("4. Biryani        - Rs.260");
        switch (item) {
            case 1: System.out.println("Ordered: Butter Chicken - Rs.220"); break;
            case 2: System.out.println("Ordered: Paneer Tikka - Rs.180"); break;
            case 3: System.out.println("Ordered: Dal Makhani - Rs.150"); break;
            case 4: System.out.println("Ordered: Biryani - Rs.260"); break;
            default: System.out.println("Invalid choice.");
        }
    }
}
